tsconfig.eslint.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src"]
```

```suggestion
  "include": ["src", "cypress"]
```

Add `cypress` folder here like in `documentation-homepage` repo?